### DISCUSSION TITLE
IServiceCollection is the right way to make ViewModels interract with each other?

### DISCUSSION THREAD

**Discussion by ahmedtalaat327** [lepoco/wpfui#1136] [Q&A]
suppose I need to change property in VM [B] from current Command in VM [A]  is this way is the RIGHT way?
            ((CLASSTYPE)(App.GetService<VIEWMODEL B>().PROPERTY B[0])).Content = "NEWVALUEFROM A";

**Comment by pomianowski** [comment]
Well, you can use https://github.com/lepoco/reflectioneventing for example 😃 Imo, VM's should not talk to each other directly

**Reply by ahmedtalaat327** [reply]
okay already implemented. 🫡
